include/functions.h

```diff
@@ -1313,46 +1313,46 @@ void func_800B1030(SkyboxContext*, f32, f32, f32);
-void func_800B31D0(UNK_ARGS);
-s32 func_800B346C(UNK_ARGS);
-void func_800B3474(UNK_ARGS);
-void func_800B34CC(UNK_ARGS);
-void func_800B34D4(UNK_ARGS);
-void func_800B34E0(UNK_ARGS);
-void func_800B3538(UNK_ARGS);
-void func_800B3560(UNK_ARGS);
-void func_800B3568(UNK_ARGS);
-void func_800B3718(UNK_ARGS);
-s32 func_800B37DC(UNK_ARGS);
-void func_800B37E4(UNK_ARGS);
-void func_800B37EC(UNK_ARGS);
+void FBFilter_InitGraphics(FBFilter* fbFilter);
+void FBFilter_InitData(FBFilter* fbFilter);
+void FBFilter_Destroy(FBFilter* fbFilter);
+FBFilter* FBFilter_Init(FBFilter* fbFilter, s32 row, s32 col);
+void FBFilter_Update(FBFilter* fbFilter);
+void FBFilter_Draw(FBFilter* fbFilter, Gfx**);
+void func_800B23E8(FBFilter* fbFilter);
+void TransitionTriforce_Start(TransitionTriforce* trnsnTriforce);
+TransitionTriforce* TransitionTriforce_Init(TransitionTriforce* trnsnTriforce);
+void TransitionTriforce_Destroy(TransitionTriforce* trnsnTriforce);
+void TransitionTriforce_Move(TransitionTriforce* trnsnTriforce, s32 updateRate);
+void TransitionTriforce_SetColor(TransitionTriforce* trnsnTriforce, u32 color);
+void TransitionTriforce_SetType(TransitionTriforce* trnsnTriforce, s32 type);
+void TransitionTriforce_Draw(TransitionTriforce* trnsnTriforce, Gfx** gfxP);
+s32 TransitionTriforce_IsDone(TransitionTriforce* trnsnTriforce);
+void TransitionWipe_Start(TransitionWipe* trnsnWipe);
+TransitionWipe* TransitionWipe_Init(TransitionWipe* trnsnWipe);
+void TransitionWipe_Destroy(TransitionWipe* trnsnWipe);
+void TransitionWipe_Move(TransitionWipe* trnsnWipe, s32 updateRate);
+void TransitionWipe_Draw(TransitionWipe* trnsnWipe, Gfx** gfxP);
+s32 TransitionWipe_IsDone(TransitionWipe* trnsnWipe);
+void TransitionWipe_SetType(TransitionWipe* trnsnWipe, s32 type);
+void TransitionWipe_SetColor(TransitionWipe* trnsnWipe, u32 color);
+void TransitionCircle_Start(TransitionCircle* trnsnCircle);
+TransitionCircle* TransitionCircle_Init(TransitionCircle* trnsnCircle);
+void TransitionCircle_Destroy(TransitionCircle* trnsnCircle);
+void TransitionCircle_Move(TransitionCircle* trnsnCircle, s32 updateRate);
+void TransitionCircle_Draw(TransitionCircle* trnsnCircle, Gfx** gfxP);
+s32 TransitionCircle_IsDone(TransitionCircle* trnsnCircle);
+void TransitionCircle_SetType(TransitionCircle* trnsnCircle, s32 type);
+void TransitionCircle_SetColor(TransitionCircle* cirtrnsnCirclecle, u32 color);
+void TransitionCircle_SetEnvColor(TransitionCircle* trnsnCircle, u32 color);
+void TransitionFade_Start(TransitionFade* trnsnFade);
+TransitionFade* TransitionFade_Init(TransitionFade* trnsnFade);
+void TransitionFade_Destroy(TransitionFade* trnsnFade);
+void TransitionFade_Move(TransitionFade* trnsnFade, s32 updateRate);
+void TransitionFade_Draw(TransitionFade* trnsnFade, Gfx** gfxP);
+s32 TransitionFade_IsDone(TransitionFade* trnsnFade);
+void TransitionFade_SetColor(TransitionFade* trnsnFade, u32 color);
+void TransitionFade_SetType(TransitionFade* trnsnFade, s32 type);
```

Comment:
I believe all these function prototypes need to be updated to have `this` for the argument, like they do in C files.